packages/evm/src/evm.ts

```diff
@@ -577,7 +575,7 @@ export class EVM implements EVMInterface {
         if (this.DEBUG) {
           debug(`Not enough gas or code size not allowed (>= Homestead)`)
         }
-        result = { ...result, ...CodesizeExceedsMaximumError(message.gasLimit) }
+        result = { ...result, ...OOGResult(message.gasLimit) }
```

Comment:
Why is this removed? The error message is handy, because then you know the reason why the contract could not be created.

Reply:
Hmm strange, I don't remember changing that. Will take a deeper look

Reply:
How come this test is not failing? https://github.com/ethereumjs/ethereumjs-monorepo/blob/f1cf6b31e6f1be3c6f71a1b8fe0696fb23cedf13/packages/evm/tests/runCall.spec.ts#L574

Reply:
Ah now I see what happened, I extracted these commits with my branch not updated with that commit: https://github.com/ethereumjs/ethereumjs-monorepo/commit/de21d490e60999f667733625a49cc72dc23f21b8 so these changes were missed

Reply:
Hmm but yeah, not sure why the test wasn't failing?

Reply:
I checked locally, it failed for me, and the CI should also pick it up (`npm run coverage`)